book/src/csi-windows.md

```diff
@@ -0,0 +1,100 @@
+# CSI Windows Suppoort
+
+## Status
+
+Status | Min K8s Version | Min CSI proxy Version | Min Node Driver Registrar Version | latest artifacts
+--|--|--
+Beta | 1.19 | 0.2.0 | 1.3.0 | [binary](gs://k8s-staging-sig-storage/csi-proxy/csi-proxy-master.exe)
```

Comment:
For the binary, is it versioned? how do we get v0.2.0 vs v0.3.0?

We also really need to get the binary promotion issue solved. Artifacts in staging are generally automatically deleted after 30 days. We can't have anyone depending on the staging location.

Reply:
Does the path include the version now?

Reply:
I would say let's not publish the staging location for now and update this once we get the official version so that we don't have users depending on a staging version that could get garbage collected.